core/nodejsActionBase/app.js

```diff
@@ -68,8 +68,56 @@ if (!platformFactory.isSupportedPlatform(targetPlatform)) {
 
 const platformImpl = factory.createPlatformImpl(targetPlatform);
 
+fs = require('fs');
+process.env.tenantId = null;
+process.env.status = 'UNIINTIALIZED'
+try {
+    if (fs.existsSync('/var/log/tenant')) {
+        const data = fs.readFileSync('/var/log/tenant', 'utf8')
+        const tenantData = JSON.parse(data)
+        if ('tenantId' in tenantData) {
+            process.env.tenantId = tenantData.tenantId
+            process.env.status = 'INITIALIZED'
+        } else {
+            //state UNIINTIALIZED
+        }
+    }
+  } catch(err) {
+    //state UNIINTIALIZED
+  }
+console.log("Status: " + process.env.status + " Tenant Id: " + process.env.tenantId)  
+
 if (typeof platformImpl !== "undefined") {
 
+    app.get('/reallocateTo/:namespace', function (req, res, next) {
+        process.env.status = 'REIINTIALIZING'
+        fs = require('fs');
+        let tenantId = req.params.namespace || false;
+        console.log("Status: " + process.env.status + " New Tenant Id: " + tenantId) 
+        const tenantData = JSON.stringify({'tenantId': tenantId})
+        if (tenantId) {
+            console.log("Tenant Data" + tenantData)
+            fs.writeFile('/var/log/tenant', tenantData, function (err, data) {
```

Comment:
I'd insert a comment to clarify that this is just to test the functionality - it's not intended to securely mutate tenants. Hardening the security will be handled separately.